### PR TITLE
Assign default value to variable passed by reference

### PR DESCRIPTION
**Pull request by sagikazarmark**
@frederikbosch can you take a look?

### CODE REVIEW

**Comment by frederikbosch** [comment]
@sagikazarmark @Nikolai-Senkevich Good catch. Can be merged after CI is OK.

File: src/Parser/IntlMoneyParser.php [hunk 1]
```diff
@@ -32,6 +32,7 @@ public function __construct(\NumberFormatter $formatter)
      */
     public function parse($money, $forceCurrency = null)
     {
+        $currency = null;
         $decimal = $this->formatter->parseCurrency($money, $currency);
```
Comment (pamil) [inline src/Parser/IntlMoneyParser.php]: Why not `$this->formatter->parse($money)` instead?

Comment (sagikazarmark) [inline src/Parser/IntlMoneyParser.php]: Because formatter is `NumberFormatter`: http://php.net/manual/en/numberformatter.parsecurrency.php

Comment (frederikbosch) [inline src/Parser/IntlMoneyParser.php]: Because `$this->formatter` is an `IntlFormatter`.

Comment (pamil) [inline src/Parser/IntlMoneyParser.php]: And it also has [`NumberFormatter::parse`](http://php.net/manual/en/numberformatter.parse.php).

Comment (pamil) [inline src/Parser/IntlMoneyParser.php]: Since we pass null as currency, it seems like a better choice.

Comment (sagikazarmark) [inline src/Parser/IntlMoneyParser.php]: But it does not give us the currency

Comment (pamil) [inline src/Parser/IntlMoneyParser.php]: Oh, I see, standard PHP library is so weird.

**Pull request merged**